app/static/js/createEvents.js

```diff
@@ -136,7 +134,23 @@ function storeMultipleOfferingEventAttributes() {
     let entries = [];
     $(".extraSlots").children().each(function(index, element) {
         let rowData = $.map($(element).find("input"), (el) => $(el).val());
+  // Initialize timepicker
```

Comment:
Indentation on this timepicker block of code is weird and the .extraSlots function does not close, causing some errors on my end.